ios/RNSScreenStackComponentView.mm

```diff
@@ -19,10 +21,14 @@ @interface RNSScreenStackComponentView () <
     UIViewControllerTransitioningDelegate>
 @end
 
+@interface RNSPanGestureRecognizer : UIPanGestureRecognizer
```

Comment:
I think you also need `@implementation RNSPanGestureRecognizes`, similar to `RNSScreenStack.m`